uk.co.compucorp.civicrm.tasksassignments/scss/civitasks/modules/_civihr-ui-select.scss

```diff
@@ -2,8 +2,8 @@
 
 
 // Include the original ui-select css + the civihr-ui-select dropdown theme
-@import "../../../../../org.civicrm.shoreditch/scss/bootstrap/vendor/ui-select/ui-select";
-@import "../../../../../civihr/org.civicrm.bootstrapcivihr/scss/components/civihr-ui-select/civihr-ui-select";
+@import '../../../../../org.civicrm.shoreditch/scss/angular/vendor/ui-select/ui-select';
```

Comment:
use `SCSSROOT`

Reply:
T&A has a compass based build system and SCSSROOT is not integrated yet, so did not add that.